compose.yml

```diff
@@ -27,6 +27,9 @@ services:
       RUST_LOG: "${RUST_LOGS:-info}"
       SEARCH_AGENT_URL: "http://host.docker.internal:5000"
       SEARCH_AGENT_MANAGER: true
+      DKN_TASKS: "${DKN_TASKS}"
```

Comment:
We should add that these 3 vars should be given in `.env` during development mode, otherwise they will default to empty string instead of being unset.

I will add it to docs after the merge.